DATA/SOLAR/ASTEROIDS/bw08_beryllium.ini

```diff
@@ -13,6 +13,9 @@ empty_cube_frequency = 0.4
 flag = rock_objects
 flag = Object_density_med
 
+[Exclusion Zones]
+exclusion = Zone_Bw08_tradelane_construct
```

Comment:
On my end this doesn't fix the asteroid issue. I found that it's caused by the fact that the specified size for the `Zone_Bw08_tradelane_construct` zone in `bw08.ini` is somehow too small. Changing that line to `size = 500, 500, 24000` does fix it. 

A bit strange though because you'd expect the previous width and height values of `400` to have an effect, but somehow the game just ignores them.

Reply:
Yes because the cubes of asterids have a size of 400 but only disappear if the zone cuts them at their center point. So there is a chance that a zone of less than 500 never cuts them on their middle point and so does not remove them properly.

Reply:
Alright. Though these three lines do not really serve any purpose since the trade lane construction zone is not actually inside the Beryllium field. Instead, the exclusion zone is already specified in the main asteroid field (`bw08_rock_asteroid_field.ini`), which is why only the size change was required to fix the problem.

Reply:
Don't be mislead. The bw08_rock_asteroid_field itself is not spawning any asteroids in the beryllium field. But the beryllium field itself still contains a part of the TLR zone. So it must contain it, or the dense beryllium field will populate the TLR zone – which it should not.

Reply:
Indeed, when looking at the *whole* zone, a part of it is inside the Beryllium field. However, the part with the trade lanes lays outside the field. Initially I believed the issue that this PR fixes was solely about the asteroids that were stuck in those trade lanes, which is why I didn't see the purpose of adding the exclusion entry to the Beryllium field. Of course the TLR exclusion zone consists of other objects like nav buoys, which are inside the Beryllium field. You could argue that a part of the rocks have not been fully removed prior to constructing the prospective trade lanes. 

After thinking about it for a bit though, I believe for consistency it is better to have the exclusion zone in the Beryllium field as well.